lib/node_modules/@stdlib/stats/base/variancech/lib/main.js

```diff
@@ -20,16 +20,32 @@
 
 // MODULES //
 
-var setReadOnly = require( '@stdlib/utils/define-nonenumerable-read-only-property' );
```

Comment:
These changes are not desired. The `main.js` file should have remained the same.